study.md

```diff
@@ -31,7 +34,13 @@ In your own words explain "this".  In your answer, be
 sure to cite any relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+
+"This" is a self-referential tool to use with objects. Its value is based on context, scope, execution sequences,
+and special commands to directly influence its value. The Javascript interpreter is always keeping track of "this"
+and the value it references.
```

Comment:
Good and when we call a method on an object `this` in the method will refer to the object we called the method on.